qiskit_nature/second_q/problems/electronic_structure_problem.py

```diff
@@ -43,6 +43,35 @@ class ElectronicStructureProblem(BaseProblem):
     The attributes `num_particles` and `num_spin_orbitals` are only available _after_ the
     `second_q_ops()` method has been called! Note, that if you do so, the method will be executed
     again when the problem is being solved.
+
+    In the fermionic case the default filter ensures that the number of particles is being
+    preserved.
+
+        .. note::
+
+            The default filter_criterion assumes a singlet spin configuration. This means, that the
+            number of alpha-spin electrons is equal to the number of beta-spin electrons.
+            If the :class:`~qiskit_nature.properties.second_quantization.electronic.AngularMomentum`
```

Comment:
Can you fix up this reference to the new place in second_q

Reply:
Done